packages/data-explorer-ui/src/components/common/Banner/banner.styles.ts

```diff
@@ -0,0 +1,9 @@
+import styled from "@emotion/styled";
+
+export const Banner = styled.div`
+  background-color: ${({ theme }) => theme.palette.primary.main};
+  color: ${({ theme }) => theme.palette.common.white};
+  padding: 8px;
+  line-height: 1;
+  text-align: center;
```

Comment:
```suggestion
```

Reply:
By removing the text alignment we give the children components complete control over how they wish to render themselves.